### ISSUE TITLE
When I watch a video and press the OK button on the remote to display the menu, there is a delay and frame skipping

### ISSUE THREAD

**Issue by tolostoi** [jurialmunkey/skin.arctic.fuse.2#513]
### Skin section

Video OSD

### Current Behavior

When I watch a video and press the OK button on the remote to display the menu, there is a delay and frame skipping. This happens with H264, H265 (maybe with other codecs too) 720p, 1080p, 4K HDR/DV/SDR ... all. Using Kodi 21 on FireTV stick 4K MAX gen 1 and gen 2, Nokia 8010 with android and Core Elec (linux). 

### Expected Behavior

When open the OSD, expecting to not disturb video playback. 

### Steps To Reproduce

1. Play Local file, or from local network (NFS, SMB) or stream ...
2. Tap on OK button to open the OSD menu

### Screenshots and Additional Info

Video: 
https://youtu.be/iTTjaqYtlgo

The video is with Arctic Horizon 2, but with Arctic Fuse 2 i the same behavior, if needed I can make video with   Arctic Fuse 2. 
With other skins (Estuary, Arctic: Zephyr-Reloaded, Confluence) there is such a problem.

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Please let me know if latest version (v2.6.7+) improves things.

**Comment by tolostoi** [comment]
> Please let me know if latest version (v2.6.7+) improves things.

With 2.6.7 on Core Elec ( nokia 8010) and Android 11 (Fire stick Max gen 2 - no change, still skip. Maybe on core elec is less noticeable. 

https://youtu.be/L8u2R2UynQY 


I think the problem is with video or opacity layers or something like that.  When open the video osd and skip, then close the osd, can open without skipping a few seconds later with no problem, but after maybe 10 seconds the problem is visible.

**Comment by Zizou3000** [comment]
I have the same problem. I tried it on shield tv pro 2019 and Homatics Box R 4K plus and it is present in both cases, it just looks like shield can  handle it better by hardware brute force.

It is really annoying and i hope you can figure it out. Thanks!

**Comment by awabnq** [comment]
I have the same problem on a Fire TV Max, have tried everything from lowering UI resolution and even playing lower resolution content but no fix.

**Comment by Aemstel2** [comment]
Same happens on my Ugoos CoreELEC box. It also used to happen on my Nvidia Shield 2019 pro. There's no such issue on a powerful Windows PC. Seems like there's something these boxes can't handle?

It's not a major problem for me, but yeah, it is a little annoying.

Edit: I was actually wrong. This used to be an issue, but I re-tested this and it's actually not happening anymore. There's no issue on the Ugoos device.

**Comment by derrgoo** [comment]
With newest version there is no delay when OSD opens. Tried on shield pro, firestick and onn. Before this there was a delay.

**Comment by xyzfre** [comment]
Working smoothly💯 no delay here as well on an Nvidia shield pro 2019.

**Comment by Zizou3000** [comment]
It is better, but not completely smooth like when you bring it up the second time in a short period of time,then it is PERFECTLY smooth. (Homatics Box R 4k plus).

**Comment by L0bsang** [comment]
Yeah, seems like the simplified initial OSD is helping avoid stutter, nothing to fetch so comes up quicker.
